src/students/entities/student.entity.ts

```diff
@@ -0,0 +1,18 @@
+import { Student } from '@prisma/client'
+
+export class StudentEntity implements Student {
+    id!: number
+    userId!: number
+    studentCode!: string
+    linkedin!: string | null
+    facebook!: string | null
+    instagram!: string | null
+    github!: string | null
+    yearOfFirstAdmission!: number
+    yearOfCompletion!: number | null
+    classOf!: string
+    createdDate!: Date
+    updatedDate!: Date
+    createdBy!: number
+    updatedBY!: number | null
```

Comment:
implement BaseDocument

Reply:
Can you explain why **base document** is needed? 
bcz all of this field is already required from 'schema.prisma`

Reply:
ok, i got it this field generate from prisma, also don't need base docs